src/js/actions/libraries.js

```diff
@@ -533,10 +534,15 @@ define(function (require, exports) {
      * @return {Promise}
      */
     var openGraphicForEdit = function (element) {
+        if (!element) {
+            log.debug("[CC Lib] openGraphicForEdit: element is missing. maybe it is already deleted.");
```

Comment:
Please add a // FIXME comment in here explaining how CEP libraries panel does it for future reference.